examples/Overlay.main.tsx

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Overlay } from '@itwin/itwinui-react';
+
+export default () => {
+  return <Overlay />;
```

Comment:
This example is empty. Update with new code.

Reply:
done